opensearch-maps-server/README.md

```diff
@@ -49,6 +50,16 @@ docker run \
     import
 ```
 
+Run the following command to import the tiles set from a local file:
+```
+docker run \
+    -e LOCAL_TILES=/tmp/planet-osm-default-z0-z8.tar.gz \
+    -v tiles-data:/usr/src/app/public/tiles/data/ \
+    -v ./planet-osm-default-z0-z8.tar.gz:/tmp/planet-osm-default-z0-z8.tar.gz \
+    opensearch/opensearch-maps-server \
+    import
```

Comment:
```suggestion
docker run \
    -v tiles-data:/usr/src/app/public/tiles/data/ \
    -v <LOCAL_ABSOLUTE_PATH_TO_TILES_DATA>:/tmp/planet-osm-default-z0-z8.tar.gz \
    opensearch/opensearch-maps-server \
    import
```

I think we need tell user which part they need to fill for the tiles local path, because there are multiple paths information in the command.

If the `LOCAL_TILES` is only used inside the container, then we don't need to expose this configuration to user and set it directly in our code.

Reply:
I see, then it makes sense to remain the environment variable `LOCAL_TILES `. My thought was that the name might confuse user to point the host machine tile data path to the `LOCAL_TILES `.